From b395e1481b31e3627a24d899f1179f87d50bfa77 Mon Sep 17 00:00:00 2001
From: Stuart Hayhurst <stuart.a.hayhurst@gmail.com>
Date: Mon, 25 Dec 2023 20:39:00 +0000
Subject: [PATCH] hwdb: Add override for headset form-factor for the Corsair
 Void Elite

Earlier versions of the headset had 'Headset' in the product name.
This version uses 'Dongle' instead, so it doesn't get given a form-factor.

(cherry picked from commit 5fd55b2c265a4df90c31081976c5bedde94baf4a)

Related: RHEL-5950
---
 hwdb.d/70-sound-card.hwdb | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/hwdb.d/70-sound-card.hwdb b/hwdb.d/70-sound-card.hwdb
index 51b235d94d..dd5ef05583 100644
--- a/hwdb.d/70-sound-card.hwdb
+++ b/hwdb.d/70-sound-card.hwdb
@@ -8,6 +8,13 @@
 #   systemd-hwdb update
 #   udevadm trigger
 
+###########################################################
+# Corsair
+###########################################################
+# Void Headsets
+usb:v1B1Cp0A51*
+ SOUND_FORM_FACTOR=headset
+
 ###########################################################
 # Steelseries
 ###########################################################
